### PR TITLE
Added a make directory function

### PR DESCRIPTION
**Pull request by tricharu**
Added the make directory features

### CODE REVIEW

File: sys_explore.py [hunk 1]
```diff
@@ -13,3 +13,6 @@
 
 #This will give the current working directory in binary
 print(os.getcwdb())
+
+#This will create a directory
```
Comment (ricky-aug) [inline sys_explore.py]: Please provide the meaningful comment

File: sys_explore.py [hunk 2]
```diff
@@ -13,3 +13,6 @@
...
+print(os.mkdir("D:\hello"))
```
Comment (ricky-aug) [inline sys_explore.py]: print spelling is wrong

File: sys_explore.py (hunk 1 above)
Comment (tricharu) [reply sys_explore.py]: Already provided the simple comment

**Comment by ricky-aug** [comment]
Merge the changes and Thanks for the help

File: sys_explore.py [hunk 3]
```diff
@@ -13,3 +13,6 @@
 
 #This will give the current working directory in binary
 print(os.getcwdb())
+
+#This will create a directory in D drive with folder name hello
```
Comment (ricky-aug) [inline sys_explore.py]: Good nice job done

**Pull request merged**